integrations/elasticsearch/src/elasticsearch_haystack/bm25_retriever.py

```diff
@@ -20,6 +48,24 @@ def __init__(
         top_k: int = 10,
         scale_score: bool = False,
     ):
+        """
+        Initialize ElasticsearchBM25Retriever with an instance ElasticsearchDocumentStore.
+
+        :param document_store: An instance of ElasticsearchDocumentStore.
+        :type document_store: ElasticsearchDocumentStore
+        :param filters: Filters applied to the retrieved Documents, for more info
+                        see `ElasticsearchDocumentStore.filter_documents`, defaults to None
```

Comment:
Sphinx should automatically generate the default value, right? (Same question everywhere else).

Reply:
As far as I know it doesn't, and I can't find anything about it either to be fair. 🤔

Reply:
I know that it has support for it having used it in [previous projects](https://curated-transformers.readthedocs.io/en/latest/decoders.html#curated_transformers.models.TransformerDecoder.forward), but it might be something that needs to be configured/theme-dependent.

If we currently always specify the default value in the docstring, we'll leave the defaults in this PR. But in that case, we should open an issue to rectify this globally.

Reply:
As of now we specify it, same thing in the Haystack core repo. I totally agree that we need more standardization on the documentation side. 

Tagging @dfokina to keep her in the loop.